rotkehlchen/chain/ethereum/decoding/votium/decoder.py

```diff
@@ -0,0 +1,67 @@
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+
+from rotkehlchen.accounting.structures import (
+    HistoryBaseEntry,
+    HistoryEventSubType,
+    HistoryEventType,
+)
+from rotkehlchen.chain.ethereum.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.ethereum.decoding.structures import ActionItem
+from rotkehlchen.chain.ethereum.structures import EthereumTxReceiptLog
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+from rotkehlchen.chain.ethereum.utils import asset_normalized_value, ethaddress_to_asset
+from rotkehlchen.types import ChecksumEthAddress, EthereumTransaction
+from rotkehlchen.user_messages import MessagesAggregator
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.ethereum.manager import EthereumManager
+
+VOTIUM_CLAIM = b'Gf\x92\x1f\\Ydm"\xd7\xd2f\xa2\x91d\xc8\xe9b6\x84\xd8\xdf\xdb\xd91s\x1d\xfd\xca\x02R8'  # noqa: E501
+VOTIUM_CONTRACT = string_to_ethereum_address('0x378Ba9B73309bE80BF4C2c027aAD799766a7ED5A')
+
+
+class VotiumDecoder(DecoderInterface):  # lgtm[py/missing-call-to-init]
+    def __init__(  # pylint: disable=super-init-not-called
+            self,
+            ethereum_manager: 'EthereumManager',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: MessagesAggregator,
+    ) -> None:
+        self.ethereum_manager = ethereum_manager
+        self.base = base_tools
+        self.msg_aggregator = msg_aggregator
+
+    def _decode_claim(  # pylint: disable=no-self-use
+        self,
+        tx_log: EthereumTxReceiptLog,
+        transaction: EthereumTransaction,  # pylint: disable=unused-argument
+        decoded_events: List[HistoryBaseEntry],
+        all_logs: List[EthereumTxReceiptLog],  # pylint: disable=unused-argument
+        action_items: Optional[List[ActionItem]],  # pylint: disable=unused-argument
+    ) -> Tuple[Optional[HistoryBaseEntry], Optional[ActionItem]]:
+        if tx_log.topics[0] != VOTIUM_CLAIM:
+            return None, None
+
+        claimed_token_address = hex_or_bytes_to_address(tx_log.topics[1])
+        claimed_token = ethaddress_to_asset(claimed_token_address)
+        if claimed_token is None:
+            return None, None
+
+        receiver = hex_or_bytes_to_address(tx_log.topics[2])
+        claimed_amount_raw = hex_or_bytes_to_int(tx_log.data[32:64])
+        amount = asset_normalized_value(amount=claimed_amount_raw, asset=claimed_token)
+
+        for event in decoded_events:
+            if event.event_type == HistoryEventType.RECEIVE and event.location_label == receiver and event.balance.amount == amount and claimed_token == event.asset:  # noqa: E501
+                event.event_subtype = HistoryEventSubType.REWARD
+                event.counterparty = 'Votium'
```

Comment:
this should be lowercase. In the frontend you also have it lowercase. Let's try to keep some rules in protocol names.

Also I plan to put them all in constants and have them somewhere gathered so we can also query which are valid values for counterparty.